pkgs/development/tools/rust/sea-orm-cli/default.nix

```
@@ -0,0 +1,28 @@
+{ fetchCrate, lib, rustPlatform, stdenv, SystemConfiguration }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sea-orm-cli";
+  version = "0.9.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-OEsUNicNRfXUctpuVAb/3Raa+x6qeg9Mtx9FtHzBcjw=";
+  };
+
+  cargoSha256 = "sha256-aiUk+3Ppn+AgezNnK7s3AlmFTduH8stg3oxHbbczBE0=";
+
+  cargoBuildFlags = [ "--bin" pname ];
+
+  buildInputs = lib.optionals stdenv.isDarwin [ SystemConfiguration ];
```

It looks like it's missing openssl for linux support.
```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [ openssl ]
    ++  lib.optionals stdenv.isDarwin [ SystemConfiguration ];
```